sources/api/schnauzer/src/helpers.rs

```diff
@@ -1301,6 +1304,37 @@ pub fn etc_hosts_entries(
     Ok(())
 }
 
+// This helper checks if any objects have '"enabled": true' in their properties.
+//
+// any_enabled takes one argument that is expected to be an array of objects,
+// then checks if any of these objects have an "enabled" property set to true.
+// If the argument is not an array of objects, does not have an "enabled" property,
+// or does not include any objects with this property set to true, it evaluates
+// to be falsey.
+handlebars_helper!(any_enabled: |arg: Value| {
+    #[derive(Deserialize)]
+    struct EnablableObject {
+        enabled: bool,
+    }
+
+    let mut result = false;
+    match arg {
+        Value::Array(items) => {
+            for item in &items {
+                let res = serde_json::from_value::<EnablableObject>(item.clone());
+                if let Ok(eo) = res {
+                    if eo.enabled {
+                        result = true;
+                        break;
+                    }
+                }
+            }
+        }
+        _ => ()
+    }
+    result
```

Comment:
Not a very important nit, take it or leave it:

```suggestion
    match arg {
        Value::Array(items) => {
            for item in &items {
                let res = serde_json::from_value::<EnablableObject>(item.clone());
                if let Ok(eo) = res {
                    if eo.enabled {
                        return true;
                    }
                }
            }
        }
        _ => ()
    }
    false
```

Reply:
If I need to touch this part again I will update, but if you don't feel strongly I'd like to avoid some rebasing and churn as I try to get everything addressed.